inventory

```diff
@@ -0,0 +1,3 @@
+[dev-machines]
```

Comment:
Plz add another inventory file with the same group and localhost and connection local set.